lib.typ

```diff
@@ -44,6 +61,7 @@
     linkedin: ("linkedin", "https://linkedin.com/in/"),
     x: ("x-twitter", "https://twitter.com/"),
     bluesky: ("bluesky", "https://bsky.app/profile/"),
+    address: ("", ""),
```

Comment:
as @Orbion-J mentioned i agree that a house fontawesome icon would fit here nicely